src/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import './App.css';
+import HomePage from './pages/HomePage';
+import CalculatorPage from './pages/CalculatorPage';
+import QuotePage from './pages/QuotePage';
+// import ToDoContainer from './components/ToDoContainer';
+import reportWebVitals from './reportWebVitals';
+// import About from './pages/About';
+import NotMatch from './pages/NotMatch';
```

Comment:
- [ ] You want to consider removing old pieces of code as comments to follow [the best practices in JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md). ♻️

Reply:
Thanks!